pkg/gardenlet/controller/shoot/shoot/reconciler_migrate.go

```diff
@@ -172,10 +172,15 @@ func (r *Reconciler) runMigrateShootFlow(ctx context.Context, o *operation.Opera
 			Fn:           flow.TaskFn(botanist.WakeUpKubeAPIServer).DoIf(wakeupRequired),
 			Dependencies: flow.NewTaskIDs(deployETCD, scaleUpETCD, waitUntilControlPlaneReady),
 		})
+		deployGardenerResourceManager = g.Add(flow.Task{
+			Name:         "Deploying gardener-resource-manager (re-run bootstrap if needed)",
```

Comment:
Nit: I would rather put the motivation in a code comment above the task and reference https://github.com/gardener/gardener/issues/7606 instead of in the task name 😄 
```suggestion
			Name:         "Deploying gardener-resource-manager",
```